drivers/volume/portworx/portworx.go

```diff
@@ -252,6 +252,21 @@ func (d *portworx) RecoverDriver(n node.Node) error {
 		return err
 	}
 
+	t = func() (interface{}, bool, error) {
+		apiNode, err := d.getClusterManager().Inspect(n.VolDriverNodeID)
```

Comment:
there was an issue where we weren't detecting the node as not ok even though it was down. Are you fixing that as part of a different change?

Reply:
@Rohit-PX let's add the storage status check too in portworx wait driver up implementation.